rmw_implementation/src/functions.cpp

```diff
@@ -119,11 +38,12 @@ get_library()
 {
   static Poco::SharedLibrary * lib = nullptr;
   if (!lib) {
-    std::string env_var = get_env_var("RMW_IMPLEMENTATION");
+    const char * cenv_var = getenv("RMW_IMPLEMENTATION");
```

Comment:
`getenv` is a platform specific function. This should use `rcutils_get_env` instead.